generate_images.js

```diff
@@ -0,0 +1,90 @@
+var Canvas = require('canvas')
+  , Image = Canvas.Image;
+var fs = require("fs");
+var path = require('path')
+
+console.log("hello");
```

Comment:
This whole file contains a lot of unnecessary logging/debug stuff. Did you mean to commit it?